docs/architecture/accounts.md

```diff
@@ -132,30 +154,35 @@ There is also a standard for a [basic fungible faucet](https://github.com/0xPoly
   ```
 </details>
 
-The contract exposes two functions `distribute` and `burn`. The first function `distribute` can only be called by the faucet owner, otherwise it fails. As inputs, the function expects everything that is needed to create a note containing the freshly minted asset, i.e., amount, [tag](https://0xpolygonmiden.github.io/miden-base/architecture/notes.html#note-metadata), [RECIPIENT](https://0xpolygonmiden.github.io/miden-base/architecture/notes.html#note-recipient).
+The contract exposes two functions `distribute` and `burn`. 
+
+The first function `distribute` can only be called by the faucet owner, otherwise it fails. As inputs, the function expects everything that is needed to create a note containing the freshly minted asset, i.e., amount, metadata, and recipient.
 
-The second function `burn` can be called by anyone to burn the tokens that are contained in a note.
+The second function `burn` burns the tokens that are contained in a note and can be called by anyone.
 
-*Info: The difference is that the `burn` procedure exposes `exec.account::incr_nonce`, so by calling `burn` the nonce of the executing account gets increased by 1 and the transaction will pass the epilogue check. The `distribute` procedure does not expose that. That means the executing user needs to call `basic::auth_tx_rpo_falcon512` which requires the private key.*
+!!! info "Difference between `burn` and `distribute`"
+    The `burn` procedure exposes `exec.account::incr_nonce`, so by calling `burn` the nonce of the executing account gets increased by 1 and the transaction will pass the epilogue check. The `distribute` procedure does not expose that. That means the executing user needs to call `basic::auth_tx_rpo_falcon512` which requires the private key.*
 
 ## Account creation
-For an account to exist it must be present in the [Account DB](https://0xpolygonmiden.github.io/miden-base/architecture/state.html#account-database) kept by the Miden node(s). However, new accounts can be created locally by users using a wallet.
 
-The process is as follows:
+For an account to exist it must be present in the [account DB](state.md#account-database) kept on the Miden node(s). 
```

Comment:
I'm always a little bit unsure with these because I don't know if it is a proper noun or a general reference. I always defer to small because that is the style guide. Capitalizing every word that "feels" important should be avoided. I'm going to expand DB to database which should fix it.